views/overlays/admin.hbs

```diff
@@ -113,6 +131,8 @@
 	socket.on('connect', function() {
 		socket.emit('request overlay');
 		//socket.emit('join', { room: 'overlay' });
+		createCharacterList('Left');
+		createCharacterList('Right');
```

Comment:
Could the creation of the selects be done outside of the socket connection event? I don't think it has any dependencies on socket data